Gopkg.lock

```diff
@@ -63,6 +121,52 @@
   pruneopts = ""
   revision = "3c58d8115a78a6879e5df75ae900846768d36895"
 
+[[projects]]
+  digest = "1:3402e0deeb94e180eaa41deceb21f9946fad1b8e0e043ea35b4e7fa962841314"
+  name = "github.com/asaskevich/govalidator"
+  packages = ["."]
+  pruneopts = ""
+  revision = "ccb8e960c48f04d6935e72476ae4a51028f9e22f"
+  version = "v9"
+
+[[projects]]
+  digest = "1:e0127006f98b92e9fd75668c139d506496da527b1b377784134ff35f01b705b0"
+  name = "github.com/aws/aws-sdk-go"
```

Comment:
Sorry, but this is no-go, we intentionally avoid AWS SDK dependency.. Why it happen to be required by azure support?